### ISSUE TITLE
Using screen reader on Awareness page

### ISSUE THREAD

**Issue by Akpos123** [Akpos123/Diamond-Testers#3]
ID: Bug-03
Title: Using Screen reader on Awareness page
Test: Use screen reader on Awareness page
Steps:
Observe: if any missing alt text for images
Expected: Images should have descriptions
Actual: Date Missing alt-text on all images
Risk Priority: low
